articles/connections/social/linkedin.md

```diff
@@ -8,67 +8,67 @@ description: This page shows you how to connect your Auth0 app to LinkedIn. You
 toc: true
 ---
 
-# Connect your app to LinkedIn
+# Connect Your App to LinkedIn
 
 To connect your Auth0 app to LinkedIn, you will need to generate a **Client ID** and **Client Secret** in a LinkedIn app, copy these keys into your Auth0 settings, and enable the connection.
 
-## 1. Log into the developer portal
+## 1. Log in to the Developer Portal
 
-Login to the [LinkedIn Developer portal](http://developer.linkedin.com/) and click **My Apps**:
+Log in to the [LinkedIn Developer portal](http://developer.linkedin.com/), and click **My Apps**:
 
 ![](/media/articles/connections/social/linkedin/linkedin-devportal-1.png)
 
-## 2. Create your app
+## 2. Create Your App
```

Comment:
We use title case for first-level headings (Connect Your App to LinkedIn) and sentence case for subheads (Create your app). See https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#formatting
We only capitalize stuff that are always capitalized, for example Lock.

I won't comment on every subhead in this PR so please go through them and undo these specific changes.